script.js

```diff
@@ -1,53 +1,154 @@
+//trouble shoot reversed os message
+var totalMatches = 0;
+var timesPlayerWon = 0;
+var numberOfDraws = 0;
+var userName = ``;
+var message;
+var normal = `normal`;
+var korean = `korean`;
+var reversed = `reversed`;
+var gameMode = normal;
+var player = `player`;
+var currentWinner = `${userName}`;
+var myOutputValue;
 var main = function (input) {
+  var helpChoose = help();
```

Comment:
Good use of a helper function. But you can consider coming up with a better name such as getElement. Try to your functions starting with a verb